env/env.sh

```diff
@@ -6,7 +6,7 @@
 ROOTD=$(cd "$(dirname "${BASH_SOURCE[0]:-$0}")/.." && pwd)
 
 if (hostname | grep -qE "\.ee\.ethz\.ch$") ;  then
-  export PULP_RUNTIME_GCC_TOOLCHAIN=/usr/pack/riscv-1.0-kgf/pulp-gcc-1.0.16
+  export PULP_RUNTIME_GCC_TOOLCHAIN=/usr/scratch/dolcedorme/smazzola/install/pulp-tnn-gnu-7.1.1
```

Comment:
Is this compiler available open-source? This is important before we merge this PR

Reply:
It is open-source, but as a fork of @da-gazzi.. we should merge it but I'm no expert here
https://github.com/da-gazzi/pulp-tnn-gnu-toolchain
As far as I could tell it's also outdated with respect to the currently used toolchain of the upstream PULP Cluster

Reply:
Let's evoke @gtagliavini for this question :D